Gemfile

```diff
@@ -76,6 +76,7 @@ gem 'strong_parameters'
 gem 'delayed_job_active_record', '0.3.3'
 gem 'daemons'
 
+gem 'rack', '~>1.4.6'
```

Comment:
@ulferts 037c722 does the same thing on the Rails 4 branch, but specifies `rack` in the _security fixes_ commented section.